test/support/utils.spec.js

```diff
@@ -0,0 +1,13 @@
+import { getNICEAccountsUrl } from "../../src/support/utils";
+
+describe("getNICEAccountsUrl", () => {
+
+	it("should return a URL with an appropriate domain", () => {
+
+		let URL = getNICEAccountsUrl("live");
+		expect(URL).toEqual("https://accounts.nice.org.uk");
+
+		URL = getNICEAccountsUrl("environment");
+		expect(URL).toEqual("https://environment-accounts.nice.org.uk");
```

Comment:
These would be better as 2 tests rather than - would give a clear intention than "appropriate domain"